composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": ">=5.4.8",
+        "php": ">=5.5.9",
         "symfony/event-dispatcher": "^2.3||^3.0",
         "symfony/options-resolver": "^2.3||^3.0",
-        "psr/http-message-implementation": "~1.0",
-        "php-http/adapter-implementation": "^0.1.0",
-        "php-http/discovery": "^0.1.1",
-        "php-http/message-decorator": "^0.1.0"
+        "php-http/client-implementation": "^1.0.0",
+        "php-http/discovery": "^0.6.0",
+        "php-http/message": "^0.2.1",
+        "puli/cli": "^1.0.0-beta8"
```

Comment:
dev dependencies are dependencies we only need for developing this library, but not necessarily when using it in a project. (not really a question of dev vs production).

i am still confused:

1. can we have full 0 config with discovery (or will have soonish?). by that i mean its enough to add fos-http-cache and a HTTPlug client/adapter to my composer dependencies and discovery + puli do their stuff "magically" without my interaction? or do i need some manual step anyways?
2. if the answer to 1. is yes, do we need the puli cli for that, or is the cli for something else?